### PR TITLE
Lps 80660

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

File: modules/sdk/gradle-plugins-workspace/build.gradle [hunk 1]
```diff
@@ -10,12 +10,12 @@ task copyGradleTestDependencies(type: Copy)
 task copyRepo(type: Copy)
 task downloadPluginsSdk(type: Download)
 
-String bundleUrl = "https://releases-cdn.liferay.com/portal/7.0.5-ga6/liferay-ce-portal-tomcat-7.0-ga6-20180320170724974.zip"
-String pluginsSdkUrl = "https://releases-cdn.liferay.com/portal/7.0.5-ga6/liferay-plugins-sdk-7.0-1.0.16-20180320170724974.zip"
+String bundleUrl = "https://releases-cdn.liferay.com/portal/7.0.6-ga7/liferay-ce-portal-tomcat-7.0-ga7-20180507111753223.zip"
```
Comment (gamerson) [inline modules/sdk/gradle-plugins-workspace/build.gradle:13]: Lets switch to the latest 7.1.0 GA version.

File: modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties [hunk 2]
```diff
@@ -56,7 +56,7 @@
     # value is the URL for Liferay Portal CE 7.0 GA7.
     #
 #if (${liferayVersion.startsWith("7.1")})
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties:58]: Lets use the latest 7.1.0 GA version.

File: modules/util/portal-tools-bundle-support/src/main/java/com/liferay/portal/tools/bundle/support/constants/BundleSupportConstants.java [hunk 3]
```diff
@@ -30,8 +30,8 @@
 	public static final String DEFAULT_BUNDLE_FORMAT = "zip";
 
 	public static final String DEFAULT_BUNDLE_URL =
-		"https://releases-cdn.liferay.com/portal/7.0.5-ga6" +
-			"/liferay-ce-portal-tomcat-7.0-ga6-20180320170724974.zip";
+		"https://releases-cdn.liferay.com/portal/7.0.6-ga7" +
```
Comment (gamerson) [inline modules/util/portal-tools-bundle-support/src/main/java/com/liferay/portal/tools/bundle/support/constants/BundleSupportConstants.java:33]: Lets use latest 7.1.0 GA

**Comment by gamerson** [comment]
@haoyisun  @jtydhr88  please update and resend, thanks!

**Comment by haoyisun** [comment]
ok

**Pull request closed without merging**